### PR TITLE
Feature/2761/null alias

### PR DESCRIPTION
**Pull request by garyluu**
For dockstore/dockstore#2761

Eagerly load a selection of endpoints...probably going to need to eagerly load more of them as new use cases come up.

Also threw in the ntp (no transfer progress) to shorten logs.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -37,18 +37,18 @@ jobs:
       - run:
           name: run the actual tests
           command: |
-            mvn --batch-mode clean install -Punit-tests
+            mvn --batch-mode clean install -Punit-tests -ntp
```
Comment (coverbeck) [inline .circleci/config.yml]: Does this require Maven 3.6.1?

Comment (garyluu) [reply .circleci/config.yml]: I believe so.

**Comment by denis-yuen** [comment]
> Also threw in the ntp (no transfer progress) to shorten logs.

:+1:

**Comment by garyluu** [comment]
> A little too broad, but the tests are a good start.
> Would prefer the use of something more focused like running `Hibernate.initialize` or similar to pull all aliases only when a single organization or workflow is pulled by ID, but not all aliases when grabbing a number of organizations or workflows.
> 
> i.e. this should also add a test to make sure that hitting https://dockstore.org/api/static/swagger-ui/index.html#/containers/allPublishedContainers doesn't grab all aliases for all published tools

I thought the intent was that it should grab all the aliases the make it searchable.

**Comment by garyluu** [comment]
Adding Louise because not sure what use case the issue is trying to solve.

**Comment by denis-yuen** [comment]
A fair question:
My response is based on the following assumptions/suggestions
1) A user wants to see the aliases for a tool/workflow when visiting that specific tool/workflow
2) There's no point in looking at all aliases for a group of tools/workflows or to re-phrase, it wouldn't be useful enough to warrant the potential performance hit
3) If the aliases are just needed to populate search (ES) to be searchable, then they should only be grabbed when entries are added to ES

**Comment by garyluu** [comment]
I assume that current organizations should be searchable by the alias which means the get-all-organizations endpoint should return all aliases. 

I also assume that we're adding something similar for my-entries.

There is currently no discernable performance impact (heavily overshadowed by eager versions)

At this point, all the major endpoints are loading it so, might as well make it eager.

**Comment by denis-yuen** [comment]
@Ldcabansay  I think we probably need more feedback on the use-case for this to figure out the optimal approach

**Comment by Ldcabansay** [comment]
1. I think the main use case is to be able to search by alias, similar to how we can search by labels. We could probably optimize just for this now.
2. I'm not sure how useful it would be in general to display to users all aliases a particular object has, but it could be useful--somewhat similar to how Zenodo shows related identifiers. Could probably implement this later.

File: dockstore-webservice/src/main/java/io/dockstore/webservice/resources/OrganizationResource.java [hunk 2]
```diff
@@ -81,7 +81,9 @@ public OrganizationResource(SessionFactory sessionFactory) {
     @UnitOfWork(readOnly = true)
     @ApiOperation(value = "List all available organizations.", notes = "NO Authentication", responseContainer = "List", response = Organization.class)
     public List<Organization> getApprovedOrganizations() {
-        return organizationDAO.findApprovedSortedByStar();
+        List<Organization> organizations = organizationDAO.findApprovedSortedByStar();
```
[the remark this answers is not in the document]
Comment (garyluu) [reply dockstore-webservice/src/main/java/io/dockstore/webservice/resources/OrganizationResource.java]: Can you elaborate?

File: dockstore-integration-testing/src/test/java/io/dockstore/webservice/SearchResourceIT.java [hunk 3]
```diff
@@ -96,6 +97,8 @@ public void testSearchOperations() throws ApiException {
         waitForRefresh(1500);
         // after publication index should include workflow
         String s = extendedGa4GhApi.toolsIndexSearch(exampleESQuery);
+        assertTrue(s.contains("\"aliases\":{}"));
```
Comment (denis-yuen) [inline dockstore-integration-testing/src/test/java/io/dockstore/webservice/SearchResourceIT.java]: very close, let's add an actual alias on workflow before `waitForRefresh` so it shows up here

**Pull request merged**